jekyll/_cci2/configuration-reference.md

```diff
@@ -316,7 +316,11 @@ Each `run` declaration represents a new shell. It's possible to specify a multi-
 
 ##### _Default shell options_
 
-Our default `shell` has a few options enabled by default:
+The default value for the `shell` option is either `/bin/bash -eo pipefail` or
+`/bin/sh -eo pipefail`, depending on whether `/bin/bash` is present or not in
+the build container. The default shell is not a login shell (`--login` or `-l`
```

Comment:
I think the following is a tad clearer: "The default value of `shell` option is `/bin/bash -eo pipefail` if `/bin/bash` is present in build container.  Otherwise it is `/bin/sh -eo pipefail`."